api-surface-swift

```diff
@@ -1,1398 +1,1281 @@
- public func startView(key: String,name: String? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopView(key: String,attributes: [AttributeKey: AttributeValue] = [:])
- public func addTiming(name: String)
- public func addError(message: String,type: String? = nil,source: RUMErrorSource = .custom,stack: String? = nil,attributes: [AttributeKey: AttributeValue] = [:],file: StaticString? = #file,line: UInt? = #line)
- public func addError(error: Error,source: RUMErrorSource = .custom,attributes: [AttributeKey: AttributeValue] = [:])
- public func startResourceLoading(resourceKey: String,request: URLRequest,attributes: [AttributeKey: AttributeValue] = [:])
- public func startResourceLoading(resourceKey: String,url: URL,attributes: [AttributeKey: AttributeValue] = [:])
- public func startResourceLoading(resourceKey: String,httpMethod: RUMMethod,urlString: String,attributes: [AttributeKey: AttributeValue] = [:])
- public func addResourceMetrics(resourceKey: String,metrics: URLSessionTaskMetrics,attributes: [AttributeKey: AttributeValue] = [:])
- public func addResourceMetrics(resourceKey: String,fetch: (start: Date, end: Date),redirection: (start: Date, end: Date)?,dns: (start: Date, end: Date)?,connect: (start: Date, end: Date)?,ssl: (start: Date, end: Date)?,firstByte: (start: Date, end: Date)?,download: (start: Date, end: Date)?,responseSize: Int64?,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopResourceLoading(resourceKey: String,response: URLResponse,size: Int64? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopResourceLoading(resourceKey: String,statusCode: Int?,kind: RUMResourceType,size: Int64? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopResourceLoadingWithError(resourceKey: String,error: Error,response: URLResponse? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopResourceLoadingWithError(resourceKey: String,errorMessage: String,type: String? = nil,response: URLResponse? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func startUserAction(type: RUMUserActionType,name: String,attributes: [AttributeKey: AttributeValue] = [:])
- public func stopUserAction(type: RUMUserActionType,name: String? = nil,attributes: [AttributeKey: AttributeValue] = [:])
- public func addUserAction(type: RUMUserActionType,name: String,attributes: [AttributeKey: AttributeValue] = [:])
- public func addAttribute(forKey key: AttributeKey, value: AttributeValue)
- public func removeAttribute(forKey key: AttributeKey)
- public static var telemetry: _TelemetryProxy
- public static func set(customVersion: String)
+# ----------------------------------
+# API surface for Datadog:
+# ----------------------------------
+
+[?] extension InternalExtension where ExtendedType: Datadog
+    public static var telemetry: _TelemetryProxy
+    public static func set(customVersion: String)
```

Comment:
The `[?]` prefix means _"we don't know if this is public symbol, but we know it contains public descendants"_.

Typically it should be fine (both compiler and linter's `lower_acl_than_parent ` rule watch it), but not always (e.g. we have some re-exports from `DatadogInternal` that are not listed here or we sometimes publicly extend `DatadogInternal` types in product modules).